tables/hbo.tbl

```diff
@@ -0,0 +1,208 @@
+letter ֣ 46-136  # munah
+letter ֤ 46-125  # mahapakh
+letter ֥ 46-134  # merkha
+letter ֦ 46-13  # merkha kefula
+letter ֧ 46-145  # darga
+letter ֨ 46-1234  # qadma
+letter ֩ 46-12345  # telisha qetana
+letter ֪ 46-13456  # yerah ben yomo
+letter ֫ 46-135  # ole
+letter ֬ 46-24  # iluy
+letter ֭ 46-24  # dehi
+letter ֮ 46-1356  # zinor
+letter ֯ 456-134  # masora circle
+letter ְ 3  # sheva
+letter ֱ 26  # hataf segol
+letter ֲ 25  # hataf patah
+letter ֳ 345  # hataf qamats
+letter ִ 24  # hiriq
+letter ֵ 34  # tsere
+letter ֶ 15  # segol
+letter ַ 14  # patah
+letter ָ 126  # qamats
+letter ֹ 456-246  # holam Malay
+letter ֺ 135  # holam haser
+letter ֻ 346  # qubuts
+letter ּ 5  # dagesh
+letter ֽ 4  # meteg
+letter ֿ 45  # rafe
+letter ׁ 456-4  # shin dot
+letter ׂ 456-45  # sin dot
+letter ׄ 456-5  # upper dot
+letter ׅ 456-56  # lower dot
+letter ׆ 456-1345  # nun hafukha
+letter ׇ 126  # qamats qatan
+letter א 1  # alef
+letter ב 1236  # bet
+letter ג 1245  # gimel
+letter ד 145  # dalet
+letter ה 125  # hey
+letter הּ 125  # hey - isolated
+letter ו 2456  # vav
+letter ז 1356  # zayin
+letter ח 1346  # het
+letter ט 2345  # tet
+letter י 245  # yod
+letter ך 16  # final kaf
+letter כ 16  # kaf
+letter ל 123  # lamed
+letter ם 134  # final mem
+letter מ 134  # mem
+letter ן 1345  # final nun
+letter נ 1345  # nun
+letter ס 234  # samekh
+letter ע 1246  # ayin
+letter ף 124  # final pe
+letter פ 1234  # Pe
+letter ץ 2346  # final tsadi
+letter צ 2346  # tsadi
+letter ק 12345  # qof
+letter ר 1235  # resh
+letter ש 456-146  # Shin
+letter װ 2456-2456  # double vav
+letter ױ 2456-245  # vav yod
+letter ײ 245-245  # double yod
+letter ׳ 456-126  # geresh
+letter ײַ 245-245-14  #  Double Yod with patah
+
+# punctuation op code characters
+punctuation ! 235  # !
+punctuation ' 3  # '
+punctuation - 36  # -
+punctuation . 256  # .
+punctuation : 25  # :
+punctuation ? 56-236  # ?
+punctuation ֑ 23  # atnokta
+punctuation ־ 36  # maqaf
+punctuation ׀ 456  # paseq
+punctuation ׃ 256  # sof pasuq
+punctuation ״ 456  # gershayim
+
+# sign op code characters
+sign # 3456  # #
+sign $ 4-234  # $
+sign % 46-356  # %
+sign & 4-12346  # &
+sign * 5-236  # nan
+sign + 5-235  # +
+sign / 5-34  # /
+sign < 4-126  # <
+sign > 4-345  # >
+sign ^ 35  # ^
+sign _ 26  # _
+sign ✔ 12356  # Check Mark Distinguishes Same kind of Varient
+sign ✕ 1235  # Mark beginning of Replacement
+sign ✖ 2456  # Mark Ends of Replacement
+sign ✗ 24  # Mark for  Insertion
+sign ✘ 35  # Marks Varients with in Apparatus
+sign ✙ 23456-6  #   Marks end of Replaced text
+sign ❁ 123-1346-1346  # Reference to LXX
+sign ❍ 135  # Marks Omitted Word
+sign ❏ 135-6  # Marks End of Omited word
+sign ❐ 1235  # Marks Replaced Word
+sign ❑ 12345  # Mark that Distinguishes between Varient
+sign ❘ 1256  # Mark that separates variations
+sign ❙ 2345  # Mark that indicates Alternative readings
+sign ❚ 14  # Mark indicates varrnt
+sign ❥ 16-6  # Marks distinguishes between varient
```

Comment:
Please point to a Unicode listing page showing where ❥ is used in the critical apparatus, because I don't see one. I only see it listed as an emoticon. I see plenty of other signs used to indicate variants and if you are missing one you will need it.